yt_dlp/postprocessor/modify_chapters.py

```diff
@@ -0,0 +1,333 @@
+import copy
+import heapq
+import os
+
+from .common import PostProcessor
+from .ffmpeg import (
+    FFmpegPostProcessor,
+    FFmpegSubtitlesConvertorPP
+)
+from .sponsorblock import SponsorBlockPP
+from ..utils import (
+    float_or_none,
+    orderedSet,
+    PostProcessingError,
+    prepend_extension,
+    traverse_obj,
+)
+
+
+_TINY_SPONSOR_OVERLAP_DURATION = 1
```

Comment:
Affects ordinary chapters too. `TINY_CHAPTER_DURATION` was a better name.

Reply:
Brought back old name.

Reply:
I thought only sponsor-sponsor overlaps were affected by this

Reply:
No. Tiny chapters present in the original list are always preserved, but if a cut or a sponsor overlap result in a creation of a tiny chapter out of a normal one, that tiny chapter will also be ignored.